.vscode/extensions.json

```
@@ -0,0 +1,16 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+  // Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "yzhang.markdown-all-in-one",
+    "DavidAnson.vscode-markdownlint",
+    "dbaeumer.vscode-eslint",
+    "ms-vsliveshare.vsliveshare",
+  ],
```

Since insomnia uses editorconfig this extension might be useful also:

```suggestion
    "editorconfig.editorconfig"
  ],
```

---

I didn't realize this was here: see https://github.com/Kong/insomnia/pull/3933